packages/pluggableWidgets/tree-view-web/src/TreeView.editorPreview.tsx

```diff
@@ -1,7 +1,67 @@
-import { ReactElement } from "react";
+import { parseStyle } from "@mendix/piw-utils-internal";
+import { GUID, WebIcon } from "mendix";
+import { createElement, ReactElement } from "react";
 import { TreeViewPreviewProps } from "../typings/TreeViewProps";
+import { TreeView } from "./components/TreeView";
 
-export function preview(_props: TreeViewPreviewProps): ReactElement | null {
-    // TODO:
-    return null;
+function mapIconToWebIcon(icon: TreeViewPreviewProps["expandIcon"] | TreeViewPreviewProps["collapseIcon"]): WebIcon {
+    if (icon) {
+        if (icon.type === "glyph") {
+            return {
+                type: "glyph",
+                iconClass: icon.iconClass
+            };
+        }
+        return {
+            type: "image",
+            iconUrl: icon.imageUrl
```

Comment:
With 9.5 the `iconUrl` property will be changed to `imageUrl`. I included tree view in [this story](https://mendix.atlassian.net/browse/WC-578) to fix it with the other widgets.